.github/styles/Papermoon/Acronyms.yml

```diff
@@ -45,10 +45,12 @@ exceptions:
   - RAM
   - REPL
   - RPC
+  - RPM
```

Comment:
Where was this acronym used? Does this mean Revolution per Minute?

Reply:
It was used here: https://github.com/polkadot-developers/polkadot-docs/pull/41#discussion_r1764021495

It's this: https://wiki.debian.org/RPM